### DISCUSSION TITLE
pake error

### DISCUSSION THREAD

**Discussion by frj9641** [tw93/Pake#699] [Q&A] [answered]
- Building app...
> pake-cli@2.3.6 build
> npm run tauri build --release
> pake-cli@2.3.6 tauri
> tauri build
error: invalid character `{` in package name: `tauri-plugin-{{name}}`, characters must be Unicode XID characters (numbers, `-`, `_`, or most letters)
 --> C:\Users\runneradmin\.cargo\git\checkouts\plugins-workspace-fd0259fcb18f51ad\5b814f5\shared\template\Cargo.toml:2:8
  |
2 | name = "tauri-plugin-{{name}}"
  |        ^^^^^^^^^^^^^^^^^^^^^^^
  |
error: failed to run custom build command for `app v0.1.0 (D:\a\Pake\Pake\node_modules\pake-cli\src-tauri)`
Caused by:
  process didn't exit successfully: `D:\a\Pake\Pake\node_modules\pake-cli\src-tauri\target\release\build\app-cde65c97616cdae7\build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=TAURI_CONFIG
  cargo:rerun-if-changed=tauri.conf.json
  cargo:rustc-cfg=desktop
  cargo:rerun-if-changed=png\dailydata_256.ico
  package.metadata does not exist
  Microsoft (R) Windows (R) Resource Compiler Version 10.0.10011.16384
  Copyright (C) Microsoft Corporation.  All rights reserved.
  D:\a\Pake\Pake\node_modules\pake-cli\src-tauri\target\release\build\app-25b9ea2d07c85df9\out\resource.rc(24) : error RC2175 : resource file \\?\D:\a\Pake\Pake\node_modules\pake-cli\icon.ico is not in 3.00 format
  --- stderr
  thread 'main' panicked at C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba1[50](https://github.com/frj9641/Pake/actions/runs/8843474563/job/24283765748#step:11:51)01f\embed-resource-2.2.0\src\windows_msvc.rs:39:13:
  RC.EXE failed to compile specified resource file
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
       Error failed to build app: failed to build app
file:///D:/a/Pake/Pake/node_modules/pake-cli/dist/cli.js:375
                reject(new Error(`${code}`));
                       ^
Error: 1
    at file:///D:/a/Pake/Pake/node_modules/pake-cli/dist/cli.js:375:24
    at D:\a\Pake\Pake\node_modules\shelljs\src\exec.js:145:9
    at ChildProcess.exithandler (node:child_process:430:5)
    at ChildProcess.emit (node:events:517:28)
    at maybeClose (node:internal/child_process:1098:16)
    at ChildProcess._handle.onexit (node:internal/child_process:303:5)
Node.js v18.20.2
mv: no such file or directory: DailyData.*
Build Success

**Comment by loctran016** [comment]
I also experience similar [issue](https://github.com/titieo/Pake/actions/runs/8963711559), it only occurs when you set icon argument (I try `"data:image/png;base64,iVBORw0KGgoAAr27Zt...U5ErkJggg=="` or `https://i.instagram.com/static/images/bloks/threads_filled_image.png/e1d87e1a9166.png`)

**Comment by tw93** [comment, marked as answer]
These two problems try to update to the latest Pake cli version, recently released version 2.3.8 In addition, it is very recommended to use the corresponding images under the corresponding system, you can refer to src-tauri/icons and png under the corresponding format.
